tests/BuildStats.Tests/PackageTests.fs

```diff
@@ -134,15 +134,22 @@ module PackageTests =
     /// -------------------------------------
 
     [<Fact>]
-    let ``NEventSocket returns correct result``() =
-        let package = MyGet.getPackageFromOfficialFeedAsync httpClient ("neventsocket-prerelease", "NEventSocket") false None |> runTask
+    let ``FSharp.Core returns correct result``() =
```

Comment:
Had to update this one because neventsocket-prerelease neventsocket _only_ has prerelease packages, so 404s without specifying the parameter as true. This _might_ be considered a breaking change, so would maybe need to make the default value for MyGet to be true if the parameter isn't specified.

Reply:
Hmm.. yeah thanks for pointing this out, but I think I'm tempted to correct it now and if someone raises a regression issue then I'll just point out to set the flag accordingly.